server/src/main/java/org/opensearch/common/util/concurrent/EWMATrackingThreadPoolExecutor.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common.util.concurrent;
+
+/**
+ * Marks the thread pool executor as supporting EWMA tracking
```

Comment:
I think we should spell out "exponential weighted moving average" somewhere.

Reply:
:+1: thx!